channels.yaml

```diff
@@ -1,6 +1,6 @@
 channels:
 - name: stable
-  latest: v1.22.9+rke2r2
+  latest: v1.22.10+rke2r2
```

Comment:
Are we still waiting for Rancher to catch up before promoting 1.23 to stable, or can we do that now since 2.6.5 is out?